handbook/engineering/cloud/manual_migrations.md

```diff
@@ -0,0 +1,32 @@
+# Cloud manual migrations
+
+Sourcegraph.com occassionally requires migrations be ran against it that are not ran against other deployments of Sourcegraph.
+
+Examples of this include:
+
+- Modifying DB users, which may not always be possible in customer environments https://github.com/sourcegraph/sourcegraph/pull/12561
+- One-off index modification tests like https://twitter.com/beyang/status/1287826896281989120 that _should_ eventually become migrations in our product.
+
+We soon want to eliminate the need for such manual migrations against prod ([tracking issue](https://github.com/sourcegraph/sourcegraph/issues/12590)), but in the meantime this document serves as a list of manual migrations that have been ran against production.
+
+## Aug 3rd, 2020 - read only user addition
+
+https://github.com/sourcegraph/sourcegraph/pull/12561
+
+Exact SQL:
+
+```sql
+TODO(efritz,chayim)
+```
+
+## July 27th, 2020 - index addition
```

Comment:
One thing that would be nice to capture in this document, maybe, is the motivation/ to what extent a command/migration is reversible or removable (say, for debugging). For example, one way to convey this for the index change is something like "if we removed this index, you'll get degraded performance when searching a lot of repos with a common prefix but other than that things will still work"

For the above read-only change, something like "we don't want to leave ourselves open to accidental changes. FOO service relies on `sgreader` so if you remove it then BAR will break" (or whatever succinct description we can give it).

Reply:
Great, makes sense!